project/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+
+  <title>Softvision Feedback App</title>
+
+  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+  
+  <link rel="stylesheet" type="text/css" href="assets/normalize.css" />
+  <link rel="stylesheet/less" type="text/css" href="assets/grid.less" />
+  <link rel="stylesheet/less" type="text/css" href="assets/common.less" />  
+  <link rel="stylesheet/less" type="text/css" href="assets/login.less" />  
+  <link rel="stylesheet/less" type="text/css" href="assets/evaluations.less" />
+  <link rel="stylesheet/less" type="text/css" href="assets/new-evaluation.less" />
+  
+  <script src="assets/less.min.js" type="text/javascript"></script>
```

Comment:
Less.min.js is imported twice; try thinking of a pattern or order for the imports, make sure they have some structure so that you can find them easier and avoid duplicate requests. Padding groups that make sense together with new lines is one way for example.

Reply:
Oh, yes.. thank you